diff --git a/grub-core/disk/cryptodisk.c b/grub-core/disk/cryptodisk.c
index 2246af51b..657b7bfe3 100644
--- a/grub-core/disk/cryptodisk.c
+++ b/grub-core/disk/cryptodisk.c
@@ -1119,7 +1119,7 @@ grub_cryptodisk_scan_device_real (const char *name,
 	/* Get the passphrase from the user, if no key data. */
 	askpass = 1;
 	part = grub_partition_get_name (source->partition);
-	grub_printf_ (N_("Enter passphrase for %s%s%s (%s): "), source->name,
+	grub_printf_ (N_("\nEnter passphrase for %s%s%s (%s): "), source->name,
 		     source->partition != NULL ? "," : "",
 		     part != NULL ? part : N_("UNKNOWN"),
 		     dev->uuid);
